storage/innobase/os/os0enc.cc

```diff
@@ -1874,7 +1894,8 @@ bool Encryption::dblwr_encrypt_page(fil_space_t *space, page_t *in_page,
   to a new memory block which is encrypted and
   the bytes will have value of length of encrypted data */
   void *in_page_before = in_page;
-  file::Block_ptr block{os_file_encrypt_page(write_request, in_page_before, &bytes)};
+  file::Block_ptr block{
+      os_file_encrypt_page(write_request, in_page_before, &bytes)};
```

Comment:
clang formatting change because of new clang version?

Reply:
yes, I think so . I have run clang-format-8 and this was in diff.